examples/deck.gl/3d-tiles/app.js

```diff
@@ -108,14 +110,7 @@ export default class App extends PureComponent {
   }
 
   async _loadExample(selectedExample) {
-    const {ionAssetId, ionAccessToken} = selectedExample;
-    if (ionAssetId && ionAccessToken) {
-      const metadata = await _getIonTilesetMetadata(ionAccessToken, ionAssetId);
-      const {url, headers} = metadata;
-      selectedExample.tilesetUrl = url;
-      selectedExample.headers = headers;
-      selectedExample.metadata = metadata;
-    }
+    selectedExample.tilesetUrl = `${TILESET_SERVER_URL}/${selectedExample.ionAssetId}/tileset.json`;
```

Comment:
- This is no longer async
- Don't mutate constants - just construct this URL in `render`